install/linux.md

```diff
@@ -5,10 +5,12 @@ title: Linux-Installation
 
 __Es müssen alle Schritte (Installieren, Testen, Aktualisieren) ausgeführt werden!__
 
-Falls man nicht am LaTeX-Kurs teilnimmt, ist TeXLive optional.
-Dann kann man allerdings kein TeX in matplotlib benutzen (führt zu weniger schönen Plots).
+Falls du nicht am LaTeX-Kurs teilnimmst, ist TeXLive optional.
+Dann kannst du allerdings kein TeX in matplotlib benutzen (führt zu weniger schönen Plots).
 
-Falls man nur am LaTeX-Kurs teilnehmen will, sollte man mindestens Visual Studio Codium (oder einen anderen Texteditor) und TeXLive installieren.
+Falls du nur am LaTeX-Kurs teilnehmen willst,
+solltest du mindestens Visual Studio Codium (oder einen anderen Texteditor)
```

Comment:
For consistency: Either use "VS Codium" here or use "Visual Studio Codium" in the macOS instructions